README.md

```diff
@@ -1,43 +1,72 @@
-# Urban Airship Segment Integration
+# Airship Segment Integration
 
 ## Setup
 
-1) Include the Urban Airship dependency in the project’s build.gradle file:
+1) Add Segment to your app as defined in the following [Segment docs](https://segment.com/docs/sources/mobile/android/).
 
+2) Include the Airship dependency in the project’s build.gradle file:
+```
     repositories {
-       ...
+        ...
 
-       maven {
-          url  "https://urbanairship.bintray.com/android"
-       }
+        maven {
+            url  "https://urbanairship.bintray.com/android"
+        }
     }
 
 
     dependencies {
-       ...
-
-       // Urban Airship SDK
-       compile 'com.urbanairship.android:segment-integration:1.0.+'
-    }
-
-
-2) Verify the applicationId is set in the project’s build.gradle file:
+        ...
 
+        // Airship SDK
+        compile 'com.urbanairship.android:segment-integration:+'
+     }
+```
 
+3) Verify the applicationId is set in the project’s build.gradle file:
+```
     android {
-       ...
+        ...
 
-       defaultConfig {
-          ...
+        defaultConfig {
+            ...
 
-          applicationId "com.example.application"
-       }
+            applicationId "com.example.application"
+        }
     }
+```
 
-
-3) Add the Urban Airship Integration factory:
-
+4) Add the Airship Integration factory:
+```
     Analytics analytics = new Analytics.Builder(context, writeKey)
-       .use(UrbanAirshipIntegration.FACTORY)
+        .use(UrbanAirshipIntegration.FACTORY)
+        ...
+        .build();
+```
+
+5) Set up a Push Provider as described in the following [Airship docs](https://docs.airship.com/platform/android/getting-started/#push-provider-setup). Add the Airship core and your chosen push provider as a dependency in your app's build.gradle file. e.g. for FCM:
+```
+   dependencies {
        ...
-       .build();
+
+       // Airship Core
+       implementation 'com.urbanairship.android:urbanairship-core:10.0.2'
```

Comment:
They shouldn't need to depend on core if they're pulling in fcm, right?

Reply:
Nope!